util/json/src/test/java/jetbrains/jetpad/json/ParsingTest.java

```diff
@@ -115,10 +115,10 @@ public void simpleString() {
   }
 
   @Test
-  public void simpleEscapedString() {
-    JsonString s = (JsonString) Json.parse("\"a\\nbc\"");
+  public void simpleEscapedChars() {
+    JsonString s = (JsonString) Json.parse("\" \\\\ \\n \\f \\t \\b \\r \"");
 
-    assertEquals("a\nbc", s.getStringValue());
+    assertEquals(" \\ \n \f \t \b \r ", s.getStringValue());
```

Comment:
This is duplicate code. You put the same string literal into the code twice. Please, consider creating a method assertParsesAndPrint(text) to do this kind of testing.